msal/src/main/java/com/microsoft/identity/client/TokenParameters.java

```diff
@@ -136,7 +136,7 @@ public ClaimsRequest getClaimsRequest() {
         return mClaimsRequest;
     }
 
-    void setAccountRecord(AccountRecord record) {
+    public void setAccountRecord(AccountRecord record) {
```

Comment:
could you update your PRs to include an overview of the changes?

Reply:
Done